core/src/main/java/nu/yona/server/messaging/service/MessageService.java

```diff
@@ -18,9 +20,9 @@
 import org.springframework.stereotype.Component;
 import org.springframework.stereotype.Service;
 
+import nu.yona.server.crypto.PublicKeyEncryptor;
```

Comment:
This knowledge should not go here but remain hidden inside the message destination.